reddit/reaper.go

```diff
@@ -49,6 +49,7 @@ type reaperImpl struct {
 }
 
 func newReaper(c reaperConfig) reaper {
+	c.client.AutoRefresh()
```

Comment:
Is this supposed to launch a concurrent process?

Reply:
Yes